03LessonThree/in-class-project/daily-planet.html

```diff
@@ -1,10 +1,49 @@
 <!DOCTYPE html>
 <html>
-    <head>
+  <head>
         <title>The Daily Planet</title>
         <link rel="stylesheet" href="css/daily-planet-style.css"/>
     </head>
     <body>
-        <!-- You code start here -->
-    </body>
-</html>
\ No newline at end of file
+      <header>
+        <img src="https://s-media-cache-ak0.pinimg.com/564x/09/3e/12/093e12c80c9402d19518408e3eba3388.jpg" alt="Image of Logo" title="Daily Planet" style="height:100px" "width:100px"/>
+        <h1>Daily Planet</h1>
+        <nav>
+          <a href="#local">Local</a>
+          <a href="#sports">Sports</a>
+          <a href="#Economy">Economy</a>
+        </nav>
+      </header>
+      <main>
+        <section>
+          <h2>Rio Olympics athletes advised to not put their heads underwater</h2>
+          <p>The water is dangerous and the world's top athletes are swimming in it. Makes you wonder what the International Olympic Committee was thinking when they arranged to have the Olympics in Rio de Janiero.</p>
+          <br>
+          <p>The Rio Olympics are kicking off on August 5th, but Rio de Janeiro’s waterways show alarming levels of pollution that could be dangerous for athletes and tourists alike. The Olympics and Paralympic aquatic venues are teeming with dangerous viruses
+            and bacteria from untreated sewage, according to a 16-month-long study commissioned by The Associated Press.</p>
+          <br>
+          <p>The findings prompted biomedical expert Valerie Harwood to give this advice to visitors, according to the AP: "Don't put your head under water."</p>
+          <p>It seems that Rio is also having to deal with Montezuma's Revenge!</p>
+          <br>
+          <br>
+        </section>
+        <aside>
+          <figure>
+            <img src="http://www.nestle-watersna.com/content/Documents/logos/low/Ozarka_Logo_LR.jpg" alt="Image of Ozarka Logo" title="Ozarka Bottled Water, Taste of the Ozarks" style="height:200px" "width:350px"/>
+            <figcaption>Ozarka Water</figcaption>
+          <p><cite>We all know Texas is a wonderful state (some might even call it great). It’s here that you’ll find some seriously wonderful spring water. Ozarka® Brand Natural Spring Water comes from springs located right here in Texas.</cite></p>
+      </figure>
+    </aside>
+    <footer>
+      <p>Copywrite Daily Planet, Inc.</p>
+      <p><a href="mailto:sampleaddress@gmail.com">Write us an Email</a></p>
+      <p>123 Fake St. Austin, TX 78747</p>
+      <a href="https://www.facebook.com/DailyPlanetShow/?fref=nf">
+        <img src="https://www.facebookbrand.com/img/fb-art.jpg" alt="Facebook logo" title="Check out our Facebook" style="height:50px" "width:50px"/></a>
+      <a href="https://twitter.com/dailyplanetdc">
+        <img src="http://seeklogo.com/images/T/twitter-logo-4BD1CE8FF3-seeklogo.com.gif" alt="Twitter logo" title="Check out our Twitter" style="height:50px" "width:50px"/></a>
+
+    </footer>
+  </main>
+  </body>
+</html>

```

Comment:
Nailed it. If you have time next time, use the dev tools to play around with section alignments so aside and article are same length.